content/en/blog/2021/announcing-notation-alpha1.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Announcing Notary v2 alpha 1
+author:  "[Steve Lasker](https://github.com/stevelasker), Microsoft Azure"
+date: 2021-10-20
+draft: false
+---
+
+We are happy to announce the [alpha 1 release][notation-release] of the [Notary v2 project][notary-v2].
+Notary v2 enables signing of all artifacts (Container Images, Software Bill of Materials, Scan Results) stored in an [OCI Distribution][oci-distribution] based registries, with [oras artifacts spec][oras-artifacts] enhancements.
+A key tenant of Notary v2 is that it enables promotion of signed artifacts within and across registries, including air-gapped and private network environments.
+
+In addition to signature promotion, Notary v2 focuses on ease of use, with minimal dependencies. While Notary v2 can integrate with other supply chain efforts, there are no additional services required to sign or validate an artifact.
+As artifacts get promoted, users/entities may add new signatures, attesting to the validity of the content for the target environment, enabling a secure supply chain workflow.
+
+Through signing, users choose the artifacts they trust, from the entities they trust, [decoupling location from identity](https://stevelasker.blog/2021/09/24/separating-identity-from-location/).
+
+![](../announcing-notation-alpha1/artifact-promotion.svg)
```

Comment:
The image has some text that is cropped in various boxes.

There is also an unreadable icon in the upper right corner.

<img width="1869" alt="Screen Shot 2021-10-21 at 1 25 53 PM" src="https://user-images.githubusercontent.com/4453979/138351832-a103f162-1565-4539-93d6-b38d36f3fad9.png">

These aren't showstopper issues, but we may want to open an issue to fix the image after the PR is merged.